sdk/core/core/CMakeLists.txt

```diff
@@ -49,6 +50,9 @@ add_executable (az_core_test test/main.c)
 
 target_link_libraries(az_core_test PRIVATE az_core)
 
+add_executable (az_auth_poc test/az_auth_poc.c)
```

Comment:
In another PR I use header files to group unit tests. IMHO, we should be consistent. We can discuss which way we would like to go.

Reply:
This is not a unit test, but an example usage. I can remove that entirely before checking in to master.

Reply:
I removed the _poc.c file, it is no longer neccessary, and updated curl_poc.c with the new way to get auth token.